docs/tutorials.rst

```diff
@@ -0,0 +1,38 @@
+Tutorials
+=========
+
+
+# TODO(zach) move the following section somewhere useful
+
+The architecture
+----------------
+
+Girder's server-side architecture is focused around the construction of RESTful
+web APIs to afford minimal coupling between the backend services and the
+frontend clients. This decoupling allows multiple clients all to use the same
+server-side interface. While Girder does contain its own single-page javascript
+web application, the system can be used by any HTTP-capable client, either inside
+or outside of the web browser environment. Girder can even be run without its
+front-end application present at all, only serving the web API routes.
+
+The web API is mostly used to interact with resources that are represented by **models**
```

Comment:
Would you consider adding something here that identifies models as [Data access objects](https://en.wikipedia.org/wiki/Data_access_object)?  This was confusing for me initially when working with girder coming from other web based systems where ```model``` referred to objects that were instantiated with the data from individual records.